src/scripts/admin/filters/button-enhanced.js

```diff
@@ -1,7 +1,7 @@
 import classnames from 'classnames';
 import iconsObj from '../utils/button-icons';
 
-const { assign, merge } = require('lodash');
+import { assign, merge } from 'lodash';
 
 const { registerBlockStyle, unregisterBlockStyle } = wp.blocks;
```

Comment:
should be loading like 
```suggestion
const { registerBlockStyle, unregisterBlockStyle } from '@wordpress/blocks';
```